airflow/providers/cncf/kubernetes/operators/job.py

```diff
@@ -284,3 +286,69 @@ def reconcile_job_specs(
             return merge_objects(base_spec, client_spec)
 
         return None
+
+
+class KubernetesPatchJobOperator(BaseOperator):
+    """
+    Update a Kubernetes Job.
+
+    .. seealso::
+        For more information on how to use this operator, take a look at the guide:
+        :ref:`howto/operator:KubernetesPatchJobOperator`
+
+    :param name: name of the Job
+    :param namespace: the namespace to run within kubernetes
+    :param body: json object with parameters for update
```

Comment:
Can you please add some description or link to documentation with the example of body we can pass here and its parameters?

Reply:
@VladaZakharova Sure, I have updated the doc string. Could you check one more time?